skypy/galaxies/tests/test_stellar_mass.py

```diff
@@ -60,3 +60,61 @@ def calc_cdf(m):
                                              size=1000, resolution=100)
     p_value = scipy.stats.kstest(sample, calc_cdf)[1]
     assert p_value >= 0.01
+
+
+def test_schechter_smf_parameters():
+    from skypy.galaxies.stellar_mass import schechter_smf_parameters
+    # Check scalar inputs
+    blue = (10**-2.423, 10**10.60, -1.21)
+    fsat_scalar, frho = 0.4, 0.2
```

Comment:
Maybe be consistent with naming convention in method.

```suggestion
    fsatellite_scalar, fenvironment = 0.4, 0.2
```